client/verta/verta/deployment/autoscaling/metrics.py

```diff
@@ -7,5 +7,63 @@
 
 @six.add_metaclass(abc.ABCMeta)
 class _AutoscalingMetric(object):
-    def _as_json(self):
+    _METRIC_ID = 0
+    _NAME = ""
+
+    def _as_dict(self):
+        return {
+            "metric_id": self._METRIC_ID,
+            "name": self._NAME,
+            "parameters": self._parameters_dict()
+        }
+
+    def _parameters_dict(self):
         raise NotImplementedError
+
+
+class CpuUtilization(_AutoscalingMetric):
+    _METRIC_ID = 1001
+    _NAME = "cpu_utilization"
+
+    def __init__(self, cpu_target):
+        self._cpu_target = cpu_target
```

Comment:
For all of these, could we also cast them to `str` for the user? I think it's reasonable for a user to expect to pass in a numerical value.

Plus, the fact that the backend wants `str`s is more of an implementation detail that the user shouldn't need to worry about (esp. if the backend's error message is that vague thing about Go type marshalling!)
```suggestion
        self._cpu_target = str(cpu_target)
```

Reply:
That makes perfect sense. Fixed.